data/mil2525d/utilities/military-features-utilities/template-gdb-toolbox/scripts/CreateTemplateGDB/UpdateDomains.py

```diff
@@ -0,0 +1,59 @@
+# ----------------------------------------------------------------------------------
```

Comment:
That's pretty standard Python practice.  It puts related python modules (.py files) together in a python package called "CreateTemplateGDB".  Since these modules are executable as standalone GP tools, or callable from each other or from someone else's python module.  It also serves as an additional namespace.

See [here](http://docs.python-guide.org/en/latest/writing/structure/) for a comprehensive discussion about this.